src/devices_models/devices/common/objective_function/market_bid.jl

```diff
@@ -275,6 +305,20 @@ end
 ######## MarketBidCost: Fixed Curves ##########
 ###############################################
 
+"""
+Check if deceremental pwl offer curve is monotonically decreasing.
+"""
+function _is_convex_decremental(pwl:: PSY.PiecewiseStepData)
```

Comment:
Ideally this function would go in the same place as the existing `is_convex`, in IS/PSY. Maybe `is_concave`?